notebooks/224-stable-diffusion-text-to-image/224-stable_dffusion-text-to-image.ipynb

```diff
@@ -0,0 +1,1271 @@
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "## login to huggingfacehub to get access to pretrained model \n",
+        "from huggingface_hub import notebook_login\n",
+        "\n",
+        "notebook_login()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Create Pytorch Models pipeline\n",
+        "StableDiffusionPipeline is an end-to-end inference pipeline that you can use to generate images from text with just a few lines of code.\n",
+        "\n",
+        "First, we load the pre-trained weights of all components of the model.\n",
+        "\n",
+        "In addition to the model id `CompVis/stable-diffusion-v1-4`, we're also passing a `use_auth_token` to the from_pretrained method `use_auth_token` to verify that you have indeed accepted the model's license."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 2,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "8NS0mazM0icN",
+        "outputId": "b98b31ca-65ca-4bb0-fad2-2599c668ccad"
+      },
+      "outputs": [
+        {
+          "data": {
+            "application/vnd.jupyter.widget-view+json": {
+              "model_id": "640f06b093964d5ab62e7e7604825c79",
+              "version_major": 2,
+              "version_minor": 0
+            },
+            "text/plain": [
+              "Fetching 19 files:   0%|          | 0/19 [00:00<?, ?it/s]"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "ftfy or spacy is not installed using BERT BasicTokenizer instead of ftfy.\n"
+          ]
+        }
+      ],
+      "source": [
+        "from diffusers import StableDiffusionPipeline\n",
+        "\n",
+        "pipe = StableDiffusionPipeline.from_pretrained(\"CompVis/stable-diffusion-v1-4\", use_auth_token=True)\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Convert models to ONNX format\n",
+        "\n",
+        "OpenVINO supports PyTorch\\* through export to the ONNX\\* format. We will use `torch.onnx.export` function for obtaining ONNX, \n",
+        "you can find more info about it in [PyTorch documentation](https://pytorch.org/docs/stable/onnx.html). \n",
+        "We need provide model object, input data for model tracing and path for model saving. \n",
+        "Optioanally, we can provide target onnx opset for conversion and other parameters specified in documentation (e.g. input and output names or dynamic shapes).\n",
+        "\n",
+        "Our model consists from 3 important parts:\n",
```

Comment:
I think it fair to say "The model consists from three important parts:"

Reply:
updated